src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -151,13 +156,9 @@ public void drive(double x, double y, double rotation) {
                 drive(ChassisSpeeds.fromFieldRelativeSpeeds(x, y, rotation, getGyroscopeRotation()));
         }
 
-        public void setRotationTarget(Rotation2d rotationTarget) {
+        public void toggleRotationTarget(Rotation2d rotationTarget) {
                 this.rotationTarget = rotationTarget;
-                forceRotationTarget = true;
-        }
-
-        public void unsetRotationTarget() {
-                forceRotationTarget = false;
+                forceRotationTarget = !forceRotationTarget;
         }
```

Comment:
I think what would help here is if we created a separate method to turn off the targetRotation. 

Something like:
```enableTargetRotation(Rotation2d rotationTarget)```
```disableTargetRotation()```